lucet-runtime/lucet-runtime-internals/src/instance/execution.rs

```diff
@@ -149,6 +149,13 @@ pub struct KillState {
     /// `tid_change_notifier` allows functions that may cause a change in `thread_id` to wait,
     /// without spinning, for the signal to be processed.
     tid_change_notifier: Condvar,
+    /// `ignore_alarm` indicates if a SIGALRM directed at this KillState's instance must be
+    /// ignored. This is necessary for a specific race where a timeout occurs right around when a
+    /// Lucet guest, or hostcall the guest made, handles some other signal: if the timeout occurs
```

Comment:
`s/timeout/termination`

```suggestion
    /// ignored. This is necessary for a specific race where a termination occurs right around when a
    /// Lucet guest, or hostcall the guest made, handles some other signal: if the termination occurs
```